File: Easylearner/src/main/kotlin/hu/onlab/easylearner/model/entities/Rating.java
package hu.onlab.easylearner.model.entities;

import lombok.Data;

import javax.persistence.*;

@Entity
@Data
@Table(name = "RATING")
public class Rating {
    @Id
    Integer Id;

    @Basic
    @Column(name = "TEACHERID")
    Integer TeacherId;

    @Basic
    @Column(name = "TOPICID")
    Integer TopicId;

    @Basic
    @Column(name = "PUNCTUALITY")
    Integer Punctiality;

    @Basic
    @Column(name = "COMMUNICATION")
    Integer Communication;

    @Basic
    @Column(name = "KNOWLEDGE")
    Integer Knowledge;

    @Basic
    @Column(name = "TEXT")
    String Text;
}

File: Android/app/src/main/java/hu/bme/aut/android/easylearner/model/Rating.kt
package hu.bme.aut.android.easylearner.model

data class Rating(
    val text : String = "",
    val communication : Int = 0,
    val punctuality : Int = 0,
    val knowledge : Int = 0,
    val teacherId : Int = 0,
    val topicId : Int = 0
){}